README.md

```diff
@@ -0,0 +1,52 @@
+# Hello-microverse
+Best pratices project and no complex coding
+
+# Getting started
+
+To get a local copy up and running, follow these steps:
+
+## Prerequisites
+
+To run this web page, you need:
+
+- A computer
+- A web browser
+
+## Setup
+
+1. Clone or download this repository to your desired folder
+2. Extract the files if necessary
+
+## Usage
+
+To run this web page:
+
+1. Go to the corresponding directory
+2. Double-click on the index.html file
+3. Select your preferred browser, if prompted
+
+Or
+
+1. Go to the corresponding directory
+2. Right-click on the index.html file
+3. Select "Open"
+4. Select your preferred browser, if prompted
+
+## Authors
+
+### Author1
+GitHub: @Matthieu96Code
+Twitter: @MatthieuSENOU
+LinkedIn: Kodjo Matthieu SENOU
+
+## Show your support
+
+If you like this project, please consider leaving a star.
+
+## Acknowledgments
+
+I would like to thank Microverse for supporting this project.
+
+## License
+
+This project is MIT licensed.
```

Comment:
- [ Required ] Please consider using the new Microverse readme template on this link [Microverse readme template >>](https://github.com/microverseinc/readme-template) for your `readme` file in this and future projects. A professional readme file sets you apart as professional in your field compared to other Job seekers, so please use [this template >>] (https://github.com/microverseinc/readme-template)

Reply:
Thank you very much for the contribution, I just thought it would be considered plagiarism if I used the template as is, I will take your comments into consideration.